lib/howl/bindings.moon

```diff
@@ -211,12 +219,17 @@ export keystrokes_for = (handler, source = nil) ->
 
   keystrokes
 
-export action_for = (translation, source='editor') ->
+export action_for = (tr, source='editor') ->
+  os = sys.info.os
+  empty = {}
```

Comment:
This use of `empty` feels odd to me. Can't the dict just be left as before like `or {}`?